.gitignore

```diff
@@ -18,3 +18,5 @@ ihatemoney/budget.db
 .python-version
 .coverage*
 prof
+*.ipynb
+*/.ipynb_checkpoints/
```

Comment:
I believe these should be better in your global `~/.gitignore` file maybe?

Reply:
didnt know that was possible!  fixed